RELEASING.md

```diff
@@ -0,0 +1,22 @@
+## Releasing a new version
+
+[This GitHub Action](.github/workflows/release.yaml) builds, and releases a new version of the wrapper to both Maven Central and Github whenever a new tag is pushed to the repository.
+
+Full release procedure is as follows:
+
+* Checkout the latest version of the `main` branch.
+* Update wrapper POM version to release (remove -SNAPSHOT)
+* Set wrapper dependency for examples to release (`examples/build.gradle`)
+* Commit / push / merge PR
+* Tag that version (eg `v0.0.5` - min `v` prefix!) with the [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and `git push origin <tagname>` 
```

Comment:
I don't understand "eg `v0.0.5` - min `v` prefix!". Release tag is literally `v0.0.5`, including `v`

Reply:
Typo - should be "mind" (without v the workflow won't run).